source/Octopus.Tentacle/Commands/RunAgentCommand.cs

```diff
@@ -53,9 +51,7 @@ public class RunAgentCommand : AbstractStandardCommand
             IWindowsLocalAdminRightsChecker windowsLocalAdminRightsChecker,
             AppVersion appVersion,
             ILogFileOnlyLogger logFileOnlyLogger,
-            Lazy<IWorkspaceCleanerTask> workspaceCleanerTask,
-            Lazy<IKubernetesPodMonitorTask> kubernetesPodMonitorTask,
-            Lazy<IKubernetesOrphanedPodCleanerTask> kubernetesOrphanedPodCleanerTask) : base(selector, log, logFileOnlyLogger)
+            IEnumerable<Lazy<IBackgroundTask>> backgroundTasks) : base(selector, log, logFileOnlyLogger)
```

Comment:
Rather than import each individual `IBackgroundTask`, we now import them all and just lazily build them and only stop if they were lazily created